src/components/dialog/content/SettingDialogContent.vue

```diff
@@ -232,14 +233,26 @@ const handleSearch = (query: string) => {
   const filteredSettings = allSettings.filter((setting) => {
     const idLower = setting.id.toLowerCase()
     const nameLower = setting.name.toLowerCase()
-    const translatedName = t(
-      `settings.${normalizeI18nKey(setting.id)}.name`
+    const translatedName = st(
```

Comment:
Switch `t` to `st` here as extension registered settings might not have translation.